### PR TITLE
telestory_cart.xml: Add info about undumped carts and fix some names (nw)

### PR DESCRIPTION
**Pull request by clawgrip**

### CODE REVIEW

**Comment by Robbbert** [comment]
Shouldn't Winie be Winnie?

**Comment by clawgrip** [comment]
Fixed, thx!

File: hash/telestory_cart.xml [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
+-Power Ranger Mystic Force - Whispering Voices / Code Busters (USA)
+-Thomas & Friends - Thomas Visits the Toy Shop / Thomas' Milkshake Mix (USA)
+-Cinderella - Cinderella / Beauties in Bloom (USA)
+-Dora the Explorer - At the Carnival / Dora Goes To School (USA)
+-Winie the Pooh - Bounce, Tigger, Bounce / Pooh's Honey Tree (USA)
+-Spongebob SquarePants - Ice-Cream Dreams / Stop the Presses! (USA)
+-The Lion King - The Lion King / The Lion King II - Simba's Pride (USA)
+-Cenerentola - Cenerentola / Cenerentola Bellezze in Fiore (Italy)
+-Il Re Reone - Il Re Leone / Il Re Leone 2: Il Regno di Simba (Italy)
+-Winnie the Pooh - Salta Tigro, salta! / Winnie the Pooh l'arsetto goloso (Italy)
```
Comment (Osso13) [inline hash/telestory_cart.xml]: arsetto should be orsetto :)

Comment (clawgrip) [reply hash/telestory_cart.xml]: Thx!!!!

**Pull request merged**